code/src/App.css

```diff
@@ -0,0 +1,449 @@
+body::after {
+  content: ""; 
+  background: url("/assets/icecream.jpg")center center;
+  opacity: 0.5;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0; 
+  position: fixed; 
+  z-index: -1;
+  background-size:cover;
+  height: 100vh;
+} 
+
+body {
+  background-color: #D1CCC1;
+}
+
+/* FORM */
+
+.form-container {
+height: 100%;
+display: flex;
+flex-direction: column;
+align-items: center;
+justify-content: space-around;
```

Comment:
the declarations should be indented :)

Reply:
You have mixed a bit with indented and not indented so just correct where its not